### PR TITLE
Update Dutch translation

### PR DESCRIPTION
**Pull request by dennisschagt**

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -1635,9 +1627,9 @@ msgid "`%s' is not a valid key command"
 msgstr "‘%s’ is een ongeldige toetsopdracht"
 
 #: src/keymap.cpp:775
-#, fuzzy, c-format
+#, c-format
 msgid "failed to parse operation sequence for %s"
-msgstr "Mislukt om wachtrij-bestand te openen (%s)"
+msgstr "mislukt om wachtrij-bestand te openen (%s)"
```
Comment (Minoru) [inline po/nl.po:1632]: "openen" and parentheses make me think that this wasn't fully updated. Can you double-check please, @dennisschagt?

**Review by Minoru — CHANGES_REQUESTED** [review]

File: po/nl.po (hunk 1 above)
Comment (dennisschagt) [reply po/nl.po]: You're right, thanks!

Fixed and force-pushed

**Comment by dennisschagt** [comment]
Sorry for the two consecutive force-pushes.
I somehow overlooked the parenthesis before the first force-push.

In case it is useful:
image

**Comment by Minoru** [comment]
Oh, it's fine, the patch is small enough that I could re-review the entire thing if I were paranoid :) Thanks for the quick update!

**Review by Minoru — APPROVED** [review]

**Pull request merged**